jobs/cf-sle12-setup/templates/pre-start

```diff
@@ -11,14 +11,14 @@ if [ ! -d $ROOTFS_DIR ]; then
   tar -pzxf $ROOTFS_PACKAGE/cf-sle12.tar.gz -C $ROOTFS_DIR
 fi
 
-rm -f $ROOTFS_DIR/usr/local/share/ca-certificates/*
+rm -rf $ROOTFS_DIR/usr/share/pki/trust/*
```

Comment:
This seems to be the correct place according to `https://github.com/openSUSE/ca-certificates`, which says:

```
Packages are expected to install their CA certificates in /usr/share/pki/trust/anchors or /usr/share/pki/trust (no extra subdir) instead of /usr/share/ca-certificates/<vendor> now. The anchors subdirectory is for regular pem files, the directory one above for pem files in openssl's 'trusted' format.
```
But I couldn't make it work on my system no matter what I tried. Are you sure that it actually works?